spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkDataWrite.java

```diff
@@ -649,6 +650,39 @@ public void testCommitUnknownException() throws IOException {
     Assert.assertEquals("Result rows should match", records, actual);
   }
 
+  @Test
+  public void testWriteWithCaseSensitiveOption() throws IOException {
```

Comment:
This test probably belongs with 
https://github.com/apache/iceberg/blob/master/spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestDataFrameWriterV2.java#L86 These tests